index.js

```diff
@@ -107,6 +108,7 @@ LFU.prototype.remove = function(key) {
 };
 
 LFU.prototype.removeFromParent = function(parent, key) {
+	Object.setPrototypeOf(parent.items, new Set());
```

Comment:
parent.items is always a set, however, if we use import, the set property is lost, so I'm recasting it into set.